composer.json

```diff
@@ -14,13 +14,35 @@
 		}
 	],
 	"require": {
-		"php": ">=7.0.0",
-		"composer/installers": "~1.0"
+		"php": ">=7.1",
+		"cakephp/cakephp": "^3.7"
+	},
+	"require-dev": {
+		"oefenweb/cakephp-codesniffer": "^3.0.0",
+		"phpmd/phpmd": "^2.6",
+		"phpstan/phpstan": "^0.10.3",
+		"phpunit/phpunit": "^5.7.14|^6.0",
+		"psy/psysh": "@stable",
+		"sebastian/phpcpd": "^3.0",
+		"dereuromark/cakephp-tools": "^1.9",
```

Comment:
Where do we use this for?

Reply:
Used by the unit tests to avoid creating wrappers